letsencrypt-apache/letsencrypt_apache/augeas_configurator.py

```diff
@@ -54,10 +54,12 @@ def check_parsing_errors(self, lens):
             lens_path = self.aug.get(path + "/lens")
             # As aug.get may return null
             if lens_path and lens in lens_path:
-                logger.error(
+                msg = (
                     "There has been an error in parsing the file (%s): %s",
                     # Strip off /augeas/files and /error
                     path[13:len(path) - 6], self.aug.get(path + "/message"))
+                logger.error(msg)
+                raise errors.PluginError(msg)
```

Comment:
Can you update docs for `interfaces.IPluginFactory` to include `:raises PluginError:`?